metron-interface/metron-alerts/src/app/alerts/alerts-list/table-view/table-view.component.spec.ts

```diff
@@ -42,17 +42,18 @@ describe('TableViewComponent', () => {
   let fixture: ComponentFixture<TableViewComponent>;
 
   beforeEach(async(() => {
-    // FIXME: mock all the unnecessary dependencies 
+    // FIXME: mock all the unnecessary dependencies
     TestBed.configureTestingModule({
-      imports: [ HttpModule ],
+      imports: [ HttpClientModule ],
       providers: [
         SearchService,
         UpdateService,
         GlobalConfigService,
         MetaAlertService,
         MetronDialogBox,
+        HttpClient,
```

Comment:
Isn't it enough to import the `HttpClientModule` only? Is it necessary to explicitly set `HttpClient` as a provider?
As far as I know, you only have to set the HttpClient when you want to set a mock instead of the original http module.

Reply:
You're right, `HttpClientModule` should be enough here. We should remove this line and the import. Thanks!